tests/functional/firefox/test_sync.py

```diff
@@ -23,6 +23,13 @@ def test_android_download_button_displayed(base_url, selenium):
 
 @pytest.mark.smoke
 @pytest.mark.nondestructive
+def test_ios_download_button_displayed(base_url, selenium):
```

Comment:
Nit: I think we can probably combine the two app store button tests into a single test, something like `test_app_store_buttons_displayed`?